### ISSUE TITLE
Docker: upgrade image to ubuntu 22.04

### ISSUE THREAD

**Issue by chevalier-m** [rundeck/rundeck#8155]
Hello,

Similar to https://github.com/rundeck/rundeck/pull/5885, I believe it is time to move the upstream docker image from Ubuntu 20.04 to the 22.04 (both being LTS).

The security scans, the adoption etc are starting to raise issues with the "old" ubuntu image. As a clear production tool, I believe rundeck should move towards the latest LTS which will soon be a year old, and has been proven stable.

**Comment by melvin-suter** [comment]
Agree. We need a newer image and we need python in it. It's bad if I have to create my own image, just to use the pre-installed winrm plugin....

**Comment by w00dst0ck** [comment]
Rundeck 5.0 container seems to be build on 

LABEL org.opencontainers.image.ref.name=ubuntu
LABEL org.opencontainers.image.version=22.04

https://hub.docker.com/layers/rundeck/rundeck/5.0.0/images/sha256-99cb24243de7537044d578986c80e4c45cff6a8c3159df15de9172063e9e3716?context=explore

**Comment by fdevans** [comment]
Thanks for the report. As part of a cleanup of our issue tracker, we're closing issues older than 12 months, in line with Rundeck's support policy of one year from a release's date (see the [release calendar](https://docs.rundeck.com/docs/history/release-calendar.html) for currently supported versions). This isn't a judgment that the report is invalid — many of these are likely still real — but we can't verify against a version this old.

If you're still seeing this on a currently supported release (6.1.x or 5.20.x+), please open a new issue using our [bug report template](https://github.com/rundeck/rundeck/issues/new?template=bug_report.yml) with your version and reproduction steps. Please don't reopen this issue — a fresh issue keeps things tracked against a supported version and gets it in front of us properly. We're actively working on improving how issues get submitted and triaged going forward.

Thanks for your patience,
The Rundeck Team
